catalog/cataloger_merge.go

```diff
@@ -58,16 +64,16 @@ func (c *cataloger) Merge(ctx context.Context, repository, leftBranch, rightBran
 		}
 
 		if message == "" {
-			message = formatMergeMessage(leftBranch, rightBranch)
+			message = formatMergeMessage(sourceBranch, destinationBranch)
 		}
 		commitID, err := c.doMergeByRelation(tx, relation, leftID, rightID, committer, message, metadata)
```

Comment:
If branches cahnged to source and destination, ID should as well

Reply:
I see the problem - the Cataloger interface defined the method with source and destination when the implementation used left and right. When I update the interface signature, it also updates the parameter names.